src/server/launchpad/index.js

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2016 Canonical Ltd.  This software is licensed under the
+ * GNU Affero General Public License version 3 (see the file LICENSE).
+ */
+
+import { conf } from '../helpers/config';
+import { Launchpad } from './client';
+
+export function getLaunchpad() {
+  return new Launchpad(
+    conf.get('LP_API_URL'), conf.get('LP_CONSUMER_KEY'),
```

Comment:
Done.  Other people can add theirs.

(The reason I only added LP_API_URL to the dev env was because that's enough for minimal anonymous access; the rest are part of actual authentication credentials and shouldn't be committed.)